C/kmsg_log.c

```diff
@@ -58,50 +58,54 @@ int klog(char *msg)
   for real "printing" to kmsg (IF LOG_LOC not defined or used)
 */
 int logwarn(char *warning)
-	{
-    char *warn = malloc(256);
-     strcpy(warn ,"Warning!: ");
-     strcat(warn ,warning);
-  	int ret = klog(warn) ;
-      return(ret);
-    free(warn);
-	}
+{
+  char *warn = malloc(256);
```

Comment:
On the github webview  the original code was written but this code is written in Atom and is presented it wisely and making me push it so I didn't get that difference